streams/src/test/java/org/apache/kafka/streams/kstream/TimeWindowsTest.java

```diff
@@ -95,27 +91,19 @@ public void advanceIntervalMustNotBeLargerThanWindowSize() {
 
     @Test
     public void gracePeriodShouldEnforceBoundaries() {
-        TimeWindows.of(ofMillis(3L)).grace(ofMillis(0L));
+        TimeWindows.ofSizeAndGrace(ofMillis(3L), ofMillis(0L));
 
         try {
-            TimeWindows.of(ofMillis(3L)).grace(ofMillis(-1L));
+            TimeWindows.ofSizeAndGrace(ofMillis(3L), ofMillis(-1L));
             fail("should not accept negatives");
         } catch (final IllegalArgumentException e) {
             //expected
         }
     }
 
-    @Test
-    public void oldAPIShouldSetDefaultGracePeriod() {
-        assertEquals(Duration.ofDays(1).toMillis(), DEPRECATED_DEFAULT_24_HR_GRACE_PERIOD);
-        assertEquals(DEPRECATED_DEFAULT_24_HR_GRACE_PERIOD - 3L, TimeWindows.of(ofMillis(3L)).gracePeriodMs());
-        assertEquals(0L, TimeWindows.of(ofMillis(DEPRECATED_DEFAULT_24_HR_GRACE_PERIOD)).gracePeriodMs());
-        assertEquals(0L, TimeWindows.of(ofMillis(DEPRECATED_DEFAULT_24_HR_GRACE_PERIOD + 1L)).gracePeriodMs());
-    }
-
```

Comment:
Same here: we should leave this test here until we remove the deprecated API. (and just suppress the warnings for only this test method)

Reply:
Updated in the latest commit